### DISCUSSION TITLE
The file wincorlib.dll indicates that it is dangerous and is suspected to be a Trojan horse.

### DISCUSSION THREAD

**Discussion by Atunala** [valinet/ExplorerPatcher#3032] [Q&A]
I don't know why it prompts, but if it is blocked by the anti-virus program, it will not work properly. I tried replacing the new wincorlib.dll to no avail.
<img width="750" alt="屏幕截图 2024-03-21 005417" src="https://github.com/valinet/ExplorerPatcher/assets/40513475/e9c84209-378e-4e65-a148-84f6a2d2eab7">

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2873

This is a false positive.

Update your windows defender signatures by going to settings -> updates & security -> windows security
Then click on virus & threat protection
Under virus & threat protection updates, click on check for updates
Then click on the button named check for updates

Then if it is still being detected as malicious, submit it as a false positive to Microsoft: https://www.microsoft.com/en-us/wdsi/filesubmission
